qa/sc_evm_rpc_web3_methods.py

```diff
@@ -36,6 +46,20 @@ def run_test(self):
         # Check that jdk version has at least 2 digits
         assert_true(re.search(r'jdk\d{2,}', clientVersion[3]))
 
+        # Check web3_sha3 method
+        self.validate(sc_node_1.rpc_web3_sha3("0x68656c6c6f20776f726c64"),
+                      "0x47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad")
+        self.validate(sc_node_1.rpc_web3_sha3("0x68656C6C6F20776F726C64"),
+                      "0x47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad")
+        self.validate(sc_node_1.rpc_web3_sha3("0x"),
+                      "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470")
+        self.validate(sc_node_1.rpc_web3_sha3("0X68656C6C6F20776F726C64"))
+        self.validate(sc_node_1.rpc_web3_sha3("0x68656c6c6f20776f726c642"))
```

Comment:
It would be nice to have some comments, explaining why some cases are not valid, at least for non obvious ones.

Reply:
Done